README.md

```diff
@@ -29,15 +29,19 @@ module "delegate_role" {
 module "agentless_scanner" {
   source = "git::https://github.com/DataDog/terraform-module-datadog-agentless-scanner"
 
-  api_key               = "YOUR API KEY"
+  api_key               = var.datadog-api-key
   instance_profile_name = module.scanner_role.instance_profile.name
 }
+
+variable "datadog-api-key" {
+
```

Comment:
@christophetd would it be possible to do a followup just to mark this example variable as:

```
variable "datadog-api-key" {
  type = string
  sensitive = true
}
```

If it makes sense to you.

Reply:
definitely good point